src/libcore/mem/mod.rs

```diff
@@ -358,12 +358,13 @@ pub fn size_of_val<T: ?Sized>(val: &T) -> usize {
 ///     - an (unstable) [extern type], then this function is always safe to
 ///       call, but may panic or otherwise return the wrong value, as the
 ///       extern type's layout is not known. This is the same behavior as
-///       [`size_of_val`] on a reference to an extern type tail.
+///       [`size_of_val`] on a reference to a type with extern type tail.
```

Comment:
```suggestion
///       [`size_of_val`] on a reference to a type with an extern type tail.
```

Reply:
Are you sure? I am not a native speaker but to me it sounds better without the article.

Reply:
I am also not a native speaker but to me `type with extern type tail` sounds wrong to me :thinking:

Reply:
Let's see what @shepmaster has to say. :)

Reply:
it sounds better with the `an` (not a native speaker as well :P )

Reply:
I'm outnumbered then, okay. :)